ScoutSuite/providers/gcp/rules/findings/cloudsql-instance-ssl-not-required.json

```diff
@@ -1,6 +1,7 @@
 {
   "dashboard_name": "Instances",
   "description": "SSL not required",
+  "Rationale": "All incoming connections to database servers should use SSL (CIS 6.1).",
```

Comment:
You're putting this in the public repo because it's a CIS rule, right?

Reply:
You implemented this in commit `d57f42331d13d4986dda8b53804098f9df35c4ec`, I just added the rationale